pkg/server/server.go

```diff
@@ -249,6 +255,36 @@ func (s *Server) Start(ctx context.Context) error {
 	}
 	logrus.WithFields(logrus.Fields{"id": s.app.ID(), "address": s.app.Address()}).Print("Started dqlite")
 
+	go func() {
+		for {
+			select {
+			case <-storageWatchTicker.C:
+				var stat unix.Statfs_t
+				unix.Statfs(s.storageDir, &stat)
+				avail := stat.Bavail * uint64(stat.Bsize)
+				// If available space is less than 10MB reject to start.
```

Comment:
This comment will quickly run out of sync - IMHO better not add the concrete threshold here.